articles/iot-hub/iot-hub-devguide-direct-methods.md

```diff
@@ -42,16 +42,18 @@ Direct methods are synchronous and either succeed or fail after the timeout peri
 
 Device method calls are HTTP-only from the cloud side, and MQTT-only from the device side.
 
+
 ## Reference topics:
 The following reference topics provide you with more information about using direct methods.
 
 ## Invoke a direct method from a back-end app
 ### Method invocation
 Direct method invocations on a device are HTTP calls which comprise:
 
-* The *URI* specific to the device (`{iot hub}/twins/{device id}/methods/`)
+* The *URI* specific to the device (`{iot hub}/twins/{device id}/methods/?api-version=2016-11-14`).
```

Comment:
Should this be: 
{iot hub}/twins/{device id}/methods?api-version=2016-11-14

i.e no / after method